yunshin/4주차/BruteForce/Beakjoon/2309_일곱난장이.js

```diff
@@ -0,0 +1,23 @@
+// DFS
+const input = require('fs').readFileSync('yunshin/4주차/BruteForce/Beakjoon/input/b_2309.txt').toString().trim().split('\n')
+const visited = new Array(9).fill(false)
+
+const result = []
+function dfs(sumNum, depth) {
+  if (depth == 7 && sumNum == 100 && result.length == 0) {
+    for (let i = 0; i < 9; i++) if (visited[i]) result.push(Number(input[i]))
+  }
+
+  for (let i = 0; i < 9; i++) {
+    if (!visited[i]) {
+      visited[i] = true
+      dfs(sumNum + Number(input[i]), depth + 1)
+      visited[i] = false
+    }
+  }
+}
+dfs(0, 0)
+result.sort((a, b) => a - b)
+result.map((item) => {
+  console.log(item)
+})
```

Comment:
map은 보통 기존 배열에 변형을 준 새로운 배열을 만들 때 사용하는 것 같아요 😀

```suggestion
console.log(result.join('\n'))
```
는 어떨까요!? join은 split과는 배열 → 문자열로 합쳐줄 수 있는 메서드입니다 :)

Reply:
아직 자바스크립트 사용법을 잘 몰라요 ㅎㅎ 좋은 정보 감사합니다.! bb